demo/admin/src/products/future/generated/ManufacturersGrid.tsx

```diff
@@ -109,16 +110,39 @@ export function ManufacturersGrid(): React.ReactElement {
     const columns: GridColDef<GQLManufacturersGridFutureFragment>[] = [
         {
             field: "id",
-            headerName: intl.formatMessage({ id: "manufacturer.id", defaultMessage: "ID" }),
+            renderHeader: () => (
+                <div style={{ display: "flex", alignItems: "center" }}>
+                    <Typography fontWeight={400} fontSize={14}>
```

Comment:
I don't think this is best practice. @jamesricky can you please suggest a better alternative?

Reply:
Custom styling should not be needed at all, when there is only a text in the header. 
So if there is no tooltip/icon, this should still use `headerName` instead of `renderHeader`. 

For when there is a tooltip/icon, the text should probably be rendered with `<GridColumnHeaderTitle label="ID" />` to make sure it looks exactly the same as the other header items, without the need for custom styling.

Reply:
> So if there is no tooltip/icon, this should still use headerName instead of renderHeader.

👍🏼